src/pages/help-centre/components/_search-result.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components'
+import { TQuestions } from '../data/_data-types'
+import SearchError from './_search-error'
+import SearchSuccess from './_search-success'
+
+type TSearchResult = {
+    has_result: boolean
```

Comment:
this seems to be a little bit confusing, can we simplify it? 
may be you can change the `has_searched_question` condition in `_search-section.tsx`?
```ts
const has_searched_question = search_value.length > 0 && searched_question.length > 0
```

Reply:
Yes sure